ios/PurchasesHybridCommon/PurchasesHybridCommonTests/PurchasesHybridAdditionsTests.swift

```diff
@@ -26,5 +26,27 @@ class PurchasesHybridAdditionsTests: QuickSpec {
                 }.notTo(raiseException())
             }
         }
+        context("configure with user defaults suite name") {
+            it("initializes without raising exceptions if no suite name is passed") {
+                expect {
+                    Purchases.configure(withAPIKey: "api key",
+                                        appUserID: nil,
+                                        observerMode: false,
+                                        userDefaultsSuiteName: nil,
+                                        platformFlavor: "hybrid-platform",
+                                        platformFlavorVersion: "1.2.3")
+                }.notTo(raiseException())
```

Comment:
couldn't come up with a better way to test than "make sure it doesn't raise exceptions", since this is a static method. let me know if you have any ideas

Reply:
What I do in Android is mock the static object and verify the methods are called with certain parameters. But I don't think you can do that in Objective-C

I don't think this `notTo(raiseException())` is bad